### ISSUE TITLE
feat: add cloud native docker image and compose example

### ISSUE THREAD

**Issue by aceforeverd** [4paradigm/OpenMLDB#1969]
**Describe the feature you'd like**

we'd like to make a new `openmldb-server` docker image, that just runs openmldb as a component. By offering different arguments into the docker from `docker run`, the server can act as component like tablet, nameserver, apiserver or taskmanager. 
Meanwhile, based on the `openmldb-server` image, we can provide docker-compose example for cluster environment or standalone environment. 

So basically there are two tasks to do:
- [ ] new `openmldb-server` docker image and new related GitHub Action workflow
- [ ] docker-compose yaml configurations for standalone and cluster mode respectively

**Additional context**

refer onebox/start_all.sh to see how to setup cluster/standalone environment in single host

related issues:
- #1464
